data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+		never
+	on enter
+		system
+			government "Gegno"
+		fail
+		"gegno: encounters" ++
+		event "gegno: suspicions"
+
+
+mission "Gegno Suspicions"
+	invisible
+	landing
+	source "Giaru Gegno"
+	to offer
+		"gegno: encounters" >= 3
+		"gegno: encounters" < 6
+		not "Gegno Genocide Defense: offered"
+	on offer
+		conversation
+			`After you go through your post-docking routine and enter the ringworld, you can't help but feel that something is a bit off. Normally the Quarg here would be swiftly walking about, focused on attending to their duties, but for some reason, there seems to be less of them around than usual. Moreover, those you do come across are occasionally having discussions with one another, sometimes looking in your direction even for the slightest second. What stands out the most is the lack of any Gegno in the area you're in at all.`
+			`	A few minutes later, while you are making your way through the pristine interior of the ring, you hear a voice call to you from down one of the various branching corridors. Turning around, you recognize the unique attire of the Quarg that has helped you in the past, along with a few others. "Circumstantial greetings once more, <first> <last>. Might we have a significant moment of your attention?"`
+			choice
+				`	"Sure."`
+					goto "follow"
+				`	"How long will it take?"`
+			`	"As long as is required, for this matter is of symbolic importance," it says, with a slightly elevated tone.`
+
+			label follow
+			`	You follow the group of Quarg to a smaller discussion room, similar to the one you've visited before. Or perhaps you've been visiting a different room each time, considering how unfathomably large a ringworld is. Once you sit down with the Quarg from before, it looks at you for a moment, looks down at the table, then back up to you.`
+			
+			branch "negative rep"
+				or
+					"reputation: Gegno Vi" < 0
+					"reputation: Gegno Scin" < 0
+
+			`	"Formerly, I have offered suggestions of heedful interplay regarding the Gegno. We have been alerted of a few additional encounters you've experienced, although they seemed minimal in nature. In such cases, it has been generally described that you left in a timely manner, in which we are notably appreciative of. However," the Quarg continues, "despite your efforts in recommended cautions, there are some meaningful figures of Gegno standing that expressed frustration in your trivial appearances. They inquired us of the possibility of unnecessary reoccurences."`
+			choice
+				`	"Why does this frustrate them?"`
+				`	"I haven't done any harm to them."`
+					goto harm
+				`	"It was more coincidential than anything."`
+					goto coincidental
+				`	"I wish to learn more about them."`
+					goto learn
+
+			label harm
+			`	"One does not need invoke violence to greatly harm another. However purest intentions may be, even the smallest unbeknownst action may rapidly tumble to one not forseenly desired." The Quarg then ponders for a moment.`
```

Comment:
Both `"Why does this frustrate them?"` and `"I haven't done any harm to them."` will fall to the "label harm" text. Is that intentional? Additionally, goto harm and label harm seem unnecessary.